Source/SwiftLintFramework/Rules/Style/SortedImportsRule.swift

```diff
@@ -1,30 +1,36 @@
 import Foundation
 import SourceKittenFramework
 
-extension Line {
-    fileprivate var contentRange: NSRange {
+fileprivate extension Line {
+    var contentRange: NSRange {
         return NSRange(location: range.location, length: content.bridge().length)
     }
 
+    // `Line` in this rule always contains word import
+    // This method returns contents of line that are before import
+    func importAttributes() -> String {
+        return content[importAttributesRange()].trimmingCharacters(in: .whitespaces)
```

Comment:
The attributes don't have to be on the same line with the import statement. They can also come in the lines above. Also, there can be multiple attributes for a single import.

Reply:
I just pushed a commit with test cases for multiple attributes on the same line, seems to work fine.

That's a great point about declaration over multiple lines though. Oof. It does appear to be an existing issue, with the default config we get this failure and correction:

```
        Example("@testable\nimport BBB\nimport ↓AAA"):
            Example("@testable\nimport AAA\nimport BBB"),
```

Do you consider this a blocking issue?

Reply:
Since it's an existing issue it shouldn't block your contribution. However, the issue can probably only be fixed by basing the rule on SwiftSyntax. With all the available test cases this is quite a fun thing to do. So in case you are interested ... another contribution is highly welcomed. 😉